setup.py

```diff
@@ -81,21 +71,35 @@
       zip_safe=False,
 
       install_requires=[
-          'setuptools',
-
+          # 'cssutils', # TODO: is this used anywhere?
```

Comment:
✂️

Reply:
Done via the GitHub interface, imma use this more now I like that you can simply make those changes 👍